examples/persistent-volume-provisioning/README.md

```diff
@@ -209,6 +209,7 @@ parameters:
     pool: kube
     userId: kube
     userSecretName: ceph-secret-user
+    imageFormat: "1"
```

Comment:
Yes, it need to be in double quotes, because `parameters` is a map of string to string.